en/after-payment/overview.md

```diff
@@ -1,32 +1,38 @@
 # Getting Notified of Payment Status
 <hr>
+You can get notifications when the payment is accepted successfully on Midtrans, so that you can process the customer order accordingly. Once the payment process is completed by the customer, Midtrans sends notification to you. Alternatively, you can retrieve the transaction status from the Midtrans Administrative Portal (MAP). You can access transaction notifications in any of the following ways.
 
-As Merchant you would like to know or get notified when a payment has been successfully accepted. So you can proceed customer order accordingly.
 
-After payment has been completed by customer (payment confirmed on Midtrans). Merchant will be notified by Midtrans, alternatively Merchant can pro-actively retrieve transaction status to Midtrans.
-
-Midtrans provides various means for merchant to obtain the transaction status:
 
 <div class="my-card">
 
-#### [A. Email Notification](/en/after-payment/email-notification.md)
-The simplest. Require no complicated set up.
+#### [Email Notification](/en/after-payment/email-notification.md)
+- You can get notified to your inbox at the event of a transaction.
```

Comment:
Can you please specify `email inbox` rather than just `inbox`, to minimize confusion